fe/fe-core/src/test/java/com/starrocks/analysis/UseStmtTest.java

```diff
@@ -17,45 +17,78 @@
 
 package com.starrocks.analysis;
 
-import com.starrocks.common.AnalysisException;
-import com.starrocks.common.UserException;
+import com.starrocks.catalog.Database;
```

Comment:
seems all the content has been changed. Better change the license header to 
```
// This file is licensed under the Elastic License 2.0. Copyright 2021-present, StarRocks Limited.
```

Reply:
done